src/Microsoft.VisualStudio.SlowCheetah.VS/Utilities/PackageUtilities.cs

```diff
@@ -179,5 +175,34 @@ public static T GetAutomationFromHierarchy<T>(IVsHierarchy pHierarchy, uint item
 
             return projectItem;
         }
+
+        private static bool IsFileTransform(string documentName, string transformName, IEnumerable<string> configs)
```

Comment:
nit: The logic at the end of this is getting a bit complex due to configs now being optional. This could be simplified to the following:

1. `IsFileTransform` can become `TryGetFileTransform` which returns true/false if one is found, and an out param of the transform name found.
2. `IsGenericFileTransform` can call it ignoring the out param, just propagating back the true/false value
3.  `IsFileTransformForBuildConfiguration` can call it and perform the additional check that the out param is within `configs`.